ReactiveCocoa/AppKit/ActionProxy.swift

```diff
@@ -0,0 +1,104 @@
+import AppKit
+import ReactiveSwift
+import enum Result.NoError
+
+internal final class ActionProxy<Owner: AnyObject> {
+	internal weak var target: AnyObject?
+	internal var action: Selector?
+	internal let signal: Signal<Owner, NoError>
+
+	private let observer: Signal<Owner, NoError>.Observer
+	private unowned let owner: Owner
+
+	internal init(owner: Owner, lifetime: Lifetime) {
+		self.owner = owner
+		(signal, observer) = Signal<Owner, NoError>.pipe()
+		lifetime.ended.observeCompleted(observer.sendCompleted)
+	}
+
+	// In AppKit, action messages always have only one parameter.
+	@objc func consume(_ sender: Any?) {
+		if let action = action {
+			if let target = target {
+				_ = target.perform(action, with: sender)
+			} else {
+				NSApp.sendAction(action, to: nil, from: sender)
```

Comment:
Can we remove the `if` and just pass `target` as `to` here?

If not, I think this deserves a comment in the code.